tools/didc/src/main.rs

```diff
@@ -205,6 +213,26 @@ fn main() -> Result<()> {
                 println!("{:?}", value);
             }
         }
+        Command::Random { annotate, format } => {
+            use candid::parser::configs::Configs;
+            use rand::Rng;
+            use serde_dhall::SimpleValue;
+            let mut rng = rand::thread_rng();
+            let seed: Vec<u8> = (0..2048).map(|_| rng.gen::<u8>()).collect();
+            let mut u = arbitrary::Unstructured::new(&seed);
+            let (env, types) = annotate.get_types(Mode::Encode)?;
+            let config = serde_dhall::from_file("random.dhall").parse::<SimpleValue>()?;
```

Comment:
What's the intended workflow here?  (The hardcoded name "random.dhall" is meant to be blessed/special, or merely provisional in this PR?)

Reply:
It's provisional. We can have a flag to read the dhall config and fall back to a default setting when not available.